lib/semantics/mod-file.cpp

```diff
@@ -730,9 +726,9 @@ static bool VerifyHeader(const char *content, std::size_t len) {
 }
 
 static std::size_t GetFileSize(const std::string &path) {
-  struct stat statbuf;
-  if (stat(path.c_str(), &statbuf) == 0) {
-    return static_cast<std::size_t>(statbuf.st_size);
+  std::size_t result;
+  if (llvm::sys::fs::file_size(path, result)) {
```

Comment:
This doesn't compile on macos because `size_t` is `unsigned long` and the 2nd argument of file_size is `uint64_t` which is `unsigned long long`. Declaring result as `uint64_t` fixes it.